Makefile

```diff
@@ -51,6 +51,10 @@ venv: requirements.txt requirements-dev.txt
 install-hooks: venv
 	$(BIN)/pre-commit install -f --install-hooks
 
+.PHONY: lint
+lint:
```

Comment:
weird, I still get it calling the install command.
![screen shot 2017-02-27 at 3 04 49 am](https://cloud.githubusercontent.com/assets/1916822/23359118/9905c4b4-fc99-11e6-887f-c24f71925816.png)
![screen shot 2017-02-27 at 3 05 52 am](https://cloud.githubusercontent.com/assets/1916822/23359136/ad0b2f3a-fc99-11e6-94e7-3d31df78ae99.png)